WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -37,7 +37,7 @@ public class PeopleViewController: UITableViewController, NSFetchedResultsContro
     }
     
     override public func tableView(tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        return CGFloat.min
+        return isViewHorizontallyCompact() ? CGFloat.min : 0
```

Comment:
Ahhhh i think we had a conflict (that method used to belong to WordPress main codebase), and @diegoreymendez had build issues.

I'll map the new method, added by Diego. Thank you!!